includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -75,7 +75,7 @@ private function getLangWithFallback( $lang ) {
 			'social-youtube' => 'https://www.youtube.com/channel/UC988qTQImTjO7lUdPfYabgQ',
 			'social-instagram' => 'https://www.instagram.com/getfandom/',
 			'app-store' => 'https://itunes.apple.com/developer/wikia-inc./id422467077',
-			'google-play' => 'https://play.google.com/store/apps/developer?id=Wikia,+Inc.',
+			'google-play' => 'https://play.google.com/store/apps/developer?id=Fandom+powered+by+Wikia',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "google-play" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)